openapi.yaml

```diff
@@ -209,9 +438,324 @@ definitions:
+  playerEmail:
+    type: string
+    example: abc@efg.com
+    format: email
+  memberExpAboveLevel:
+    type: integer
+    minimum: 0
+    description: Player's experience point above current level.
+    example: 1234
+  playerId:
+    type: string
+    example: '101'
+  playerIds:
+    type: array
+    items:
+      $ref: '#/definitions/playerId'
+  playerInGamePatchBody:
+    properties:
+      data:
+        type: object
+        properties:
+          id:
+            $ref: '#/definitions/playerId'
+          type:
+            $ref: '#/definitions/playerType'
+          attributes:
+            type: object
+            properties:
+              playerBet:
+                $ref: '#/definitions/playerBet'
+              playerStatus:
+                $ref: '#/definitions/playerStatus'
+              playerCards:
+                $ref: '#/definitions/cards'
+            additionalProperties: false
+            required:
+              - id
+              - type
+        additionalProperties: false
+    required:
+      - data
+    additionalProperties: false
+  playerInGameResource:
+    properties:
+      id:
+        $ref: '#/definitions/playerId'
+      type:
+        $ref: '#/definitions/playerType'
+      attributes:
+        type: object
+        properties:
+          memberName:
+            $ref: '#/definitions/memberName'
+          memberLevel:
+            $ref: '#/definitions/memberLevel'
+          memberExpAboveLevel:
+            $ref: '#/definitions/memberExpAboveLevel'
+          playerBet:
+            $ref: '#/definitions/playerBet'
+          playerStatus:
+            $ref: '#/definitions/playerStatus'
+          playerCards:
+            $ref: '#/definitions/cards'
+  playerInGameResult:
+    properties:
+      data:
+        $ref: '#/definitions/playerInGameResource'
+  playerInGameResults:
+    properties:
+      data:
+        type: array
+        items:
+          $ref: '#/definitions/playerInGameResource'
+  memberLevel:
+    type: integer
+    minimum: 1
+    description: Player's level in the system.
+    example: 20
+  memberName:
+    type: string
+    example: Jack
+  playerPassword:
+    type: string
+    example:  hunter2
+    format: password
+  memberPatchBody:
+    properties:
+      data:
+        type: object
+        properties:
+          id:
+            $ref: '#/definitions/playerId'
+          type:
+            $ref: '#/definitions/playerType'
+          attributes:
+            type: object
+            properties:
+              memberName:
+                $ref: '#/definitions/memberName'
+              playerEmail:
+                $ref: '#/definitions/playerEmail'
+              playerPassword:
+                $ref: '#/definitions/playerPassword'
+              memberLevel:
+                $ref: '#/definitions/memberLevel'
+              memberExpAboveLevel:
+                $ref: '#/definitions/memberExpAboveLevel'
+              gameIds:
+                $ref: '#/definitions/gameIds'
```

Comment:
These fields in the patch body should match the field names in the `member` resource